docs/Listener.md

```diff
@@ -0,0 +1,29 @@
+
+# Kyma Listener
+
+Listener component that listens to events sent by the Kyma [watcher](https://github.com/kyma-project/kyma-watcher) component.
+
+## Overview
+
+The listener module is typically used with operators built using kube-builder but its use is not resticted only to that.
+### Use
+
```

Comment:
Are the first two steps optional, or why do they contain "might"?
Is the only mandatory step the third step?

Reply:
All the steps are mandatory. Should I remove “might” from the first two?

Reply:
I've added two suggestions! :)

Reply:
Thank you! Applied them 👍